addons/account/migrations/9.0.1.1/post-migration.py

```diff
@@ -990,7 +990,7 @@ def fast_compute_cash_basis(env):
                         WHEN 0.0 = ROUND(
                             SUM(ABS(COALESCE(aml.debit, 0.0) - COALESCE(aml.credit, 0.0))),
                             CASE
-                                WHEN MAX(COALESCE(rc.rounding, 0)) BETWEEN 0 AND 1
+                                WHEN MAX(rc.rounding) < 1
```

Comment:
If you don't check this is bigger than zero, you could fall in a division by zero error, not?

Reply:
That would trigger a division by zero indeed. But is `0.0` a valid value for currency rounding? It is not a value that is present in a regular database (and default for new currencies is 0.01)

```
odoo/openerp/addons/base/res$ grep rounding res_currency_data.xml  |sort|uniq
            <field name="rounding">0.0001</field>
            <field name="rounding">0.01</field>
            <field name="rounding">1.00</field>
            <field name="rounding">1</field>
```